classes/04class/exercises/c04-iac01/frdvo/iam.tf

```diff
@@ -0,0 +1,50 @@
+resource "aws_iam_role" "ec2-read-s3-role-tf2" {
+  name = "ec2-read-s3-role-tf2"
+
+  assume_role_policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Action": "sts:AssumeRole",
+      "Principal": {
+        "Service": "ec2.amazonaws.com"
+      },
+      "Effect": "Allow",
+      "Sid": ""
+    }
+  ]
+}
```

Comment:
tip: template_file might help in the future https://registry.terraform.io/providers/hashicorp/template/latest/docs/data-sources/file